contrib/build-linux/pkg-debian/DEBIAN/prerm

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+rm /usr/bin/ZeronetBrowser
```

Comment:
I assume this wasn't intentional, since it's not related to Electrum?

Reply:
Indeed, my bad I have done it for another project and did a copy/past. This has been corrected now. Thanks